StoreApp.BLL/Services/ProductService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Interfaces;
+using StoreApp.DAL.Entities;
+using StoreApp.DAL.Intefaces;
+using System.Collections.Generic;
+
+namespace StoreApp.BLL.Services
+{
+    class ProductService : IProductService
+    {
+        IUnitOfWork DataBase;
+        public ProductService(IUnitOfWork uof)
+        {
+            DataBase = uof;
+        }
```

Comment:
The methods must be splitted by using indents

Reply:
fixed